proposed/event-dispatcher-meta.md

```diff
@@ -37,85 +37,84 @@ The Working Group identified four possible workflows for event passing, based on
 * Collection.  ("Give me all your things, that I may do something with that list.")
 * Alternative chain.  ("Here's a thing; the first one of you that can handle it do so, then stop.")
 
-On further review, it was determined that Collection was a special case of Object enhancement (the collection being the object that is enhanced).  Alternative chain is similarly a special case of Object enhancement, as the signature is identical and the dispatch workflow is nearly identical.  That leaves two relevant workflows:
+On further review, it was determined that:
```

Comment:
This is a passive sentence. I suggest rephrasing it to: "On further review, the working group determined that:"

Reply:
Updated.